feat: add filter to check invalid chars in user input (#5227)
codeigniter4/CodeIgniter4: user_guide_src/source/incoming/filters.rst

Provided Filters
****************

> These filters are bundled with CodeIgniter4: ``Honeypot``, ``CSRF``, ``DebugToolbar`` and ``InvalidChars``.

Reviewer: ```suggestion
The filters bundled with CodeIgniter4 are: ``Honeypot``, ``CSRF``, ``DebugToolbar``, and ``InvalidChars``.
```

> .. note:: The filters are executed in the declared order  that is defined in the config file, but there is one exception to this and it concerns the ``DebugToolbar``, which is always executed last. This is because ``DebugToolbar`` should be able to register everything that happens in other filters.

Reviewer: Although not part of the PR but still..
```suggestion
.. note:: The filters are executed in the order defined in the config file. However, if enabled, ``DebugToolbar`` is always executed last because it should be able to capture everything that happens in the other filters.
```
